test/tests/tsdx-build-closure-compiler.test.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @jest-environment node
+ */
+
+const shell = require('shelljs');
+const util = require('../fixtures/util');
+
+shell.config.silent = false;
+
+const grep = (pattern, file) => {
+  const output = shell.grep(pattern, file);
```

Comment:
@agilgur5  unfortunately shelljs.grep output.code always 0 for matched/unmatched patterns, so need to test  output.stdout

Reply:
https://github.com/shelljs/shelljs/blob/a8452b89f2b9dd372da81e973a16074a5628b0b6/src/grep.js#L73

Reply:
Oh, welp, guess my tests in #627 don't completely work either then. Actual `grep` does have an error code when it doesn't match, that's a really confusing mismatch from `shelljs` 😕

Seems like that line doesn't throw because it's silent. `common.error` seems to throw when not silent.

Probably want to put this helper into `util` since it'll be used by other tests like in #627

Reply:
sure, done